feat(serverless): autoscaling concept (#4312)
scaleway/docs-content: pages/serverless-containers/concepts.mdx

  - serverless
---

## Autoscaling

> Autoscaling refers to the ability of Serverless Containers to automatically adjust the number of intances without manual intervention.

Reviewer: ```suggestion
Autoscaling refers to the ability of Serverless Containers to automatically adjust the number of instances without manual intervention.
```

Scaling mechanisms ensure that resources are provisioned dynamically to handle incoming requests efficiently while minimizing idle capacity and cost.

> Autoscaling parameters are [min-scale](/serverless-containers/concepts/#min-scale) and [max-scale](/serverless-containers/concepts/#max-scale), scaling policies available are:

Reviewer: ```suggestion
Autoscaling parameters are [min-scale](/serverless-containers/concepts/#min-scale) and [max-scale](/serverless-containers/concepts/#max-scale). Available scaling policies are:
```

> Autoscaling parameters are [min-scale](/serverless-containers/concepts/#min-scale) and [max-scale](/docs/serverless-containers/concepts/#max-scale), scaling policies available are:

Reviewer 2: ```suggestion
Autoscaling parameters are [min-scale](/serverless-containers/concepts/#min-scale) and [max-scale](/serverless-containers/concepts/#max-scale), scaling policies available are:
```

> * **Concurrent requests:** requests incoming on the ressource at the same time. Default value suitable for most use cases.

Reviewer: ```suggestion
* **Concurrent requests:** requests incoming to the resource at the same time. Default value suitable for most use cases.
```
